style/servo/selector_parser.rs

```diff
@@ -224,7 +231,8 @@ impl PseudoElement {
             },
             PseudoElement::Backdrop |
             PseudoElement::DetailsSummary |
-            PseudoElement::Marker  => PseudoElementCascadeType::Lazy,
+            PseudoElement::Marker |
+            PseudoElement::ColorSwatch => PseudoElementCascadeType::Lazy,
```

Comment:
Please keep these in alphabetical order.

Reply:
Done, please recheck!